jboss-negotiation-toolkit/.classpath

```diff
@@ -21,6 +21,7 @@
 	<classpathentry kind="con" path="org.eclipse.m2e.MAVEN2_CLASSPATH_CONTAINER">
 		<attributes>
 			<attribute name="maven.pomderived" value="true"/>
+			<attribute name="org.eclipse.jst.component.dependency" value="/WEB-INF/lib"/>
```

Comment:
I would suggest a PR like this should avoid the Eclipse updates - really we should probably remove them entirely from the project and add to the .gitignore as it distracts from the real purpose of the PR